compiler/src/iree/compiler/Codegen/Dialect/CPU/IR/IREECPUAttrs.cpp

```diff
@@ -158,6 +158,24 @@ Attribute LoweringConfigAttr::getTilingLevelAttr(MLIRContext *ctx,
       ctx, tileSizes, /*interchange=*/{}, scalableFlags);
 }
 
+SmallVector<LoweringConfigLevelInfo>
+LoweringConfigAttr::getAvailableTilingInfo() {
+  SmallVector<LoweringConfigLevelInfo> result;
+  for (int i = 0, e = TilingLevel::MaxNumTileLevels; i < e; ++i) {
```

Comment:
```suggestion
  for (unsigned i = 0, e = TilingLevel::MaxNumTileLevels; i < e; ++i) {
```

Reply:
Isn't it an unsigned underlying type though: https://github.com/iree-org/iree/blob/81bbe6794bcc824acb64555bdaa9d27eda82c0fc/compiler/src/iree/compiler/Codegen/Dialect/CPU/IR/IREECPUTypes.h#L24C1-L34C1?

Reply:
Thanks for pointing out, I missed it. The other issue is that I thought it is `enum class`. I'll fix them in a follow-up. I switched to unsigned for this one.